### ISSUE TITLE
Individual TAB string colours

### ISSUE THREAD

**Issue by yonah-ag** [Jojo-Schmitz/MuseScore#134]
### Your idea

Allow the user to define the colour of strings on an individual basis.

### Problem to be solved

TAB for a standard guitar has no centre string as a frame of reference so it is easy to mis-read strings 3 and 4. Allowing users to define the colour would provide a useful visual reference. (Additionally it could be used to make the TAB stave look more like a guitar fretboard).

### Additional context

My workaround is to add a coloured line overlay with the correct Z-order but it's a bit painstaking since this can only be done manually and not via a plugin.

Example: https://musescore.com/user/28842914/scores/7342763

**Comment by Jojo-Schmitz** [comment]
First and foremotst you shpould request such feature in the upstream repository.
If and when it got implemented there, I'd be looking it backporting it, but I'd also consider PRs from others. I won't be coding new features myself though (unless I personally have an urgent need ;-))

**Comment by yonah-ag** [comment]
Thanks for the feedback.
I don't know what the upstream repository is but I will have a go at making this enhancement myself in 3.7.

**Comment by Jojo-Schmitz** [comment]
See https://github.com/musescore/MuseScore/issues/new/choose

**Comment by yonah-ag** [comment]
That's the page I used to submit this request.

**Comment by Jojo-Schmitz** [comment]
No, you used https://github.com/Jojo-Schmitz/MuseScore/issues/new/choose

**Comment by yonah-ag** [comment]
Got it. I wasn't even thinking MS4. I did try MS4 but there were no improvements for TABbers and too many missing features. I'll see about coding this enhancement in 3.7.

**Comment by Jojo-Schmitz** [comment]
I see.
